### PR TITLE
check the length if kubeVer for maybe panic

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
check the length if kubeVer for maybe panic

### CODE REVIEW

File: cmd/poseidon/poseidon.go [hunk 1]
```diff
@@ -121,11 +121,14 @@ func main() {
...
 	}
 	defer conn.Close()
-	// Check if frimament grpc service is available and then proceed
+	// Check if firmament grpc service is available and then proceed
 	WaitForFirmamentSerive(fc)
 	go schedule(fc)
 	go stats.StartgRPCStatsServer(statsServerAddress, firmamentAddress)
 	kubeVer := strings.Split(kubeVersion, ".")
+	if len(kubeVer) < 2 {
+		glog.Fatalf("Incorrect content in --kubeVersion %s", kubeVersion)
```
Comment (m1093782566) [inline cmd/poseidon/poseidon.go]: add hint message? E.g. kubeVersion should be in the format of X.Y?

Comment (hanxiaoshuai) [reply cmd/poseidon/poseidon.go]: Done

**Comment by m1093782566** [comment]
/lgtm

@shivramsrivastava PTAL.

File: cmd/poseidon/poseidon.go [hunk 2]
```diff
@@ -121,11 +121,14 @@ func main() {
...
 	WaitForFirmamentSerive(fc)
 	go schedule(fc)
 	go stats.StartgRPCStatsServer(statsServerAddress, firmamentAddress)
 	kubeVer := strings.Split(kubeVersion, ".")
+	if len(kubeVer) < 2 {
+		glog.Fatalf("Incorrect content in --kubeVersion %s, kubeVersion should be in the format of X.Y", kubeVersion)
+	}
 	kubeMajorVer, err := strconv.Atoi(kubeVer[0])
 	if err != nil {
 		glog.Fatalf("Incorrect content in --kubeVersion %s", kubeVersion)
```
Comment (m1093782566) [inline cmd/poseidon/poseidon.go]: BTW, can you add detailed message here and below as well?

Comment (hanxiaoshuai) [reply cmd/poseidon/poseidon.go]: Sure， Done

File: cmd/poseidon/poseidon.go [hunk 3]
```diff
@@ -121,18 +121,21 @@ func main() {
...
 	go schedule(fc)
 	go stats.StartgRPCStatsServer(statsServerAddress, firmamentAddress)
 	kubeVer := strings.Split(kubeVersion, ".")
+	if len(kubeVer) < 2 {
+		glog.Fatalf("Incorrect content in --kubeVersion %s, kubeVersion should be in the format of X.Y", kubeVersion)
+	}
 	kubeMajorVer, err := strconv.Atoi(kubeVer[0])
 	if err != nil {
-		glog.Fatalf("Incorrect content in --kubeVersion %s", kubeVersion)
+		glog.Fatalf("Incorrect content in --kubeVersion %s, kubeVersion should be in the format of X.Y and X should be integer", kubeVersion)
```
Comment (m1093782566) [inline cmd/poseidon/poseidon.go]: ...X should be an non-negative integer.

Comment (m1093782566) [reply cmd/poseidon/poseidon.go]: Same below.

Comment (hanxiaoshuai) [reply cmd/poseidon/poseidon.go]: Done

**Comment by m1093782566** [comment]
/lgtm

THANKS!

**Comment by hanxiaoshuai** [comment]
cc @shivramsrivastava Done

**Comment by shivramsrivastava** [comment]
@hanxiaoshuai Thanks.

/lgtm

**Pull request merged**